mongodbatlas/resource_mongodbatlas_advanced_cluster.go

```diff
@@ -540,6 +579,22 @@ func resourceMongoDBAtlasAdvancedClusterUpdate(ctx context.Context, d *schema.Re
 		}
 	}
 
+	/*
+		Check if advanced configuration option has a changes to update it
```

Comment:
I think this comment is not clear. Maybe something like "update advanced configuration options if needed" would be better

Reply:
Understood, will update the comment, thank you